addons/im_odoo_support/static/src/js/im_odoo_support.js

```diff
@@ -44,6 +44,9 @@
             }
         },
         start_support: function(){
+            // The free livechat service doesn't exist anymore
+            window.open('https://www.odoo.com/help', '_blank');
+            return;
```

Comment:
:-/ so maybe change the manifest to `'auto_install': False` and add a comment "deprecated" in the description or something